rustls-libssl/src/lib.rs

```diff
@@ -615,6 +616,13 @@ impl SslContext {
         self.verify_x509_store.pointer()
     }
 
+    fn set_x509_store(&mut self, store: *mut X509_STORE) {
+        if store.is_null() {
+            return;
+        }
```

Comment:
Looking at this again I think this should be adjusted to allow clearing the `X509_Store` (probably by constructing a `OwnedX509Store::Default()` for `NULL`. I will fix.

Reply:
Fixed.